docs/installation.rst

```diff
@@ -8,38 +8,37 @@ Installation
 
 .. code-block:: bash
 
-    $ git clone https://github.com/BeyondTheClouds/enos
-
-You should also choose to go with a virtualenv. Please refer to the `virtualenv
-<https://virtualenv.pypa.io/en/stable/>`_ documentation for further information.
+    $ pip install enos
```

Comment:
I have had a note line 35. I propose to edit the `reservation.yaml.sample` after branching stable/ocata